jl4-decision-service/src/Backend/Jl4.hs

```diff
@@ -182,9 +187,9 @@ mkBoolean b =
     True -> l4True
     False -> l4False
 
-mkNumericLit :: Int -> Lit
+mkNumericLit :: Real a => a -> Lit
 mkNumericLit =
-  NumericLit emptyAnno
+  NumericLit emptyAnno . toRational
```

Comment:
why this typeclass overloading? Doesn't that mean that if we come from e.g. `Double`, we already lost the precision we opted into `Rational` for?